cockatrice/src/window_main.cpp

```diff
@@ -319,6 +319,9 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
         case Response::RespClientIdRequired:
             QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\nPlease close and reopen your client to try again."));
             break;
+        case Response::RespContextError:
+            QMessageBox::critical(this, tr("Error"), tr("An internal error has occurred, please try closing and reopening your client and try again. If the error persists try updating your client to the most recent build and if need be contact your software provider."));
```

Comment:
We shouldn't have those kind of updates. Did we?

Generally adding a new proto item or adding a new field to an existing item is safe. It's only really unsafe if you remove existing fields or messages.